Modules/Timer/Timer/Project/Views/ProjectViewController.swift

```diff
@@ -34,14 +34,26 @@ public class ProjectViewController: UIViewController, Storyboarded {
     public override func viewDidLoad() {
         super.viewDidLoad()
 
-        store.select({ $0.editableProject })
-            .map({ $0?.name })
+        // Project name
+        store.compactSelect({ $0.editableProject })
+            .map({ $0.name })
```

Comment:
I think this reads better
```suggestion
        store.compactSelect({ $0.editableProject?.name })
```